circle.yml

```diff
@@ -6,13 +6,7 @@ test:
     - NODE_ENV=test make client/config/index.js:
         parallel: true
   override:
-    - bin/run-lint :
-        parallel: true
```

Comment:
The [files modifier](https://circleci.com/docs/parallel-manual-setup/#using-configuration-files-modifier) accepts only globs, so we cannot use them to write git expressions. We could find a way to set parallelism up again [by using env variables](https://circleci.com/docs/parallel-manual-setup/#env-splitting) in `run-lint` script, though.

The reason I was OK switching off parallelism and not pursuing the second approach was: if we are going to run `eslint` only for the files modified in the branch, I guess they are a handful, not a lot. Parallelism, then, is not going to accelerate things that much and, in return, we have a clearer and more understandable solution (without that logic in the script). My assumptions about the size and number of files by PR may be totally wrong as I am still not very familiar with the common practices around here - and so the gains of parallelization in this specific case could be bigger than I thought. What do you think?